AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -2100,7 +2100,7 @@ reischmann.biz###global-loader-layer-background
 twitch.tv##.follow-prompt
 twitch.tv###twilight-sticky-footer-root
 av-kappa.com,avokazu.com###bott0m0verlay
-thebaffler.com##.boxzilla-word-factory-subscribe-fullprice
+thebaffler.com#$#.boxzilla-word-factory-subscribe-fullprice { overflow: auto !important; padding-right: 0 !important; }
```

Comment:
damn, wrong tag

Reply:
You can try to merge the RR into one commit with the "rebase" button.